x-pack/plugins/actions/server/actions_client.ts

```diff
@@ -598,17 +600,8 @@ export class ActionsClient {
       })
     );
 
-    try {
-      await this.connectorTokenClient.deleteConnectorTokens({ connectorId: id });
-    } catch (error) {
-      this.auditLogger?.log(
-        connectorAuditEvent({
-          action: ConnectorAuditAction.DELETE,
-          savedObject: { type: 'action', id },
-          error,
-        })
-      );
-    }
+    await this.connectorTokenClient.deleteConnectorTokens({ connectorId: id });
```

Comment:
The try/catch block used to swallow the error so that inability to delete the associated tokens wouldn't block the deletion of the connector. Do we want errors deleting the connector token to be thrown?

Reply:
Good catch. I think catching and logging will be ok in this case, keep the functionality the same as it was. I will update.